### PR TITLE
docs: add feature_toggle HOWTO

### PR DESCRIPTION
**Pull request by pshiu**
## Description

Add a HOWTO guide that walks through how to use waffle flags in frontend-app-payment, and how it was used for the rollout of the updated Stripe Payment Intents payment processor in frontend-app-payment and ecommerce.

## Supporting information

* Jira: [REV-3150](https://2u-internal.atlassian.net/browse/REV-3150) Update the decisions docs with Stripe work for payment MFE and ecommerce.

## Checklist
- [X] Consider PCI compliance impact and whether this PR changes how credit card information is handled.
- [X] Intend to [release and monitor](https://openedx.atlassian.net/wiki/spaces/RS/pages/1835106870/How+to+contribute+to+our+repositories) this PR promptly after merge.

### CODE REVIEW

File: docs/how_tos/feature_toggle.rst [hunk 1]
```diff
@@ -0,0 +1,120 @@
+Feature Toggle HOWTO
+====================
+
+
+Introduction
+------------
+
+This HOWTO explains how frontend-app-payment can use Waffle Flags for feature toggling.
```
Comment (julianajlk) [inline docs/how_tos/feature_toggle.rst:8]: [nit] Specify as Django Waffle Flags?

Comment (julianajlk) [reply docs/how_tos/feature_toggle.rst:8]: Nervermind, you explain that soon after!

Comment (julianajlk) [reply docs/how_tos/feature_toggle.rst:8]: Do we ever explain why we use waffle flags instead of frontend feature flags via configs?

**Review by julianajlk — APPROVED** [review]
This is so useful, will be a great reference for future waffle flags 👏

File: docs/how_tos/feature_toggle.rst (hunk 1 above)
Comment (pshiu) [reply docs/how_tos/feature_toggle.rst:8]: No, we don't! Good idea, I added it. Let me know what you think. [[diff](https://github.com/openedx/frontend-app-payment/compare/6995d4bbf0f7600fbd0b0f3254659f488e464136..728ce1ec32acf343c92b0edaaf78a51bf34dfec3)]

Comment (julianajlk) [reply docs/how_tos/feature_toggle.rst:8]: Thank you, looks great!  I would add too that you'd need to merge/deploy the frontend if you take that route (if it's not obvious?).

Comment (pshiu) [reply docs/how_tos/feature_toggle.rst:8]: Good add. [[diff](https://github.com/openedx/frontend-app-payment/compare/728ce1ec32acf343c92b0edaaf78a51bf34dfec3..a503a1afd812781cc5fa8769b433bde08e9c0615?short_path=c354c72#diff-c354c72baf6f4b76a4c28f36f23eaadc8fc34e5782d7349e2a1d6a2acb0c77ce)]

**Pull request merged**